src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -96,12 +96,15 @@ Not all [permissions](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExte
 | `proxy`                                                                                                                        | Control browser proxy settings                                                                                                                                                                                   |
 | `sessions`                                                                                                                     | Access recently closed tabs                                                                                                                                                                                      |
 | `tabs`                                                                                                                         | Access browser tabs                                                                                                                                                                                              |
+| `tabHide`                                                                                                                      | Hide and show browser tabs                                                                                                                                                                                           |
 | `topSites`                                                                                                                     | Access browsing history                                                                                                                                                                                          |
-| `webNavigation`                                                                                                                | Access browser activity during navigation                                                                                                                                                                        |
-| Clipboard access:                                                                                                                |
+| `trialML`                                                                                                                      | Download and run AI models on your device                                                                                                                                                                            |
```

Comment:
The `trialML` API is currently Nightly-only. Should we document this here?

There is no MDN documentation, only https://firefox-source-docs.mozilla.org/toolkit/components/ml/extensions.html

Reply:
[I suggested](https://github.com/mozilla/extension-workshop/pull/2013#discussion_r1958795239) including it here. My intent was to encourage developers to experiment with the API and share feedback, but I see how this so could lead to confusion. 

I'm leaning more towards including it, but somehow clarifying that this is an experimental or non-stable capability. To float a couple of ideas, we could do that by appending `**Experimental:**` at the start of the description, adding a new column to identify stable/experimental, or move experimental permissions to a separate table.

Reply:
```suggestion
```